spec/features/schools/dashboard/manage_cip_training_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+require_relative "./manage_training_steps"
+
+RSpec.describe "Manage CIP training", js: true, with_feature_flags: { induction_tutor_manage_participants: "active" } do
+  include ManageTrainingSteps
+
+  scenario "CIP Induction Mentor without materials chosen" do
+    given_there_is_a_school_that_has_chosen_cip_for_2021
```

Comment:
Looks like we're missing some A11y checks?

Reply:
Now updated 👍